src/index.html

```diff
@@ -17,13 +17,13 @@ <h2 class="h2">Recent scores</h2>
                     <button type="button" class="refresh-btn">Refresh</button>
                 </div>
                 <ul class="scores-list">
-                    <li class="score-item ash">John: 100</li>
+                    <!-- <li class="score-item ash">John: 100</li>
                     <li class="score-item">Zoe: 150</li>
                     <li class="score-item ash">Natasha: 55</li>
                     <li class="score-item">Peter: 78</li>
                     <li class="score-item ash">Chloe: 42</li>
                     <li class="score-item">Zerubabel: 190</li>
-                    <li class="score-item ash">Freya: 200</li>
+                    <li class="score-item ash">Freya: 200</li> -->
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.

Reply:
@Dipeshtwis thank you. That was an oversight 🤦‍♂️